### ISSUE TITLE
[App Configuration] Update the configs for downloading specific resources from Composition Resource

### ISSUE THREAD

**Issue by qaziabubakar-vd** [opensrp/fhircore#2022]
**Name of feature to enhance**
Update the local configs for downloading the configs for downloading specific resources from Composition Resource
Linked Issue: https://github.com/opensrp/fhircore/issues/1790 

**Description of feature**

**Describe the enhancement**

**Comment by pld** [comment]
Can you explain this issue in more detail, it's not clear to me what the task is here

**Comment by FikriMilano** [comment]
@qaziabubakar-vd is this the ticket where we sync Questionnaires in the Composition resource when in debug mode?

**Comment by FikriMilano** [comment]
I think this functionality disappeared in the last few refactors

**Comment by qaziabubakar-vd** [comment]
@FikriMilano Yes, I tested this on staging server on "demo" app, was working fine their.
I will check again

**Comment by qaziabubakar-vd** [comment]
@pld I'll update the ticket and add more details.

**Comment by dubdabasoduba** [comment]
> Can you explain this issue in more detail, it's not clear to me what the task is here

@pld this issue is to make sure the local test app id also uses the composition locally provide composition resource to download the required content.
